### PR TITLE
Fixed classic render not rendering (rebased)

### PR DESCRIPTION
**Pull request by kgd-al**
Follow up from #559 with rebase

### CODE REVIEW

File: simulation/revolve2/simulation/simulator/_viewer.py
Comment (oliverweissl) [inline simulation/revolve2/simulation/simulator/_viewer.py:1]: this indeed is nicer if gone

**Review by oliverweissl — APPROVED** [review]

**Comment by oliverweissl** [comment]
@kgd-al  please do a Squash & Merge and remove all comments from the push :)

**Comment by kgd-al** [comment]
@oliverweissl Isn't that supposed to be done from your side?

**Comment by oliverweissl** [comment]
i dont mind :D

**Pull request merged**